package.json

```diff
@@ -40,31 +40,31 @@
   },
   "homepage": "https://github.com/agenda/agenda-rest#README",
   "devDependencies": {
-    "@babel/cli": "^7.0.0-beta.40",
-    "@babel/core": "^7.0.0-beta.40",
-    "@babel/preset-env": "^7.0.0-beta.40",
-    "@babel/register": "^7.0.0-beta.40",
+    "@babel/cli": "^7.0.0-beta.55",
```

Comment:
They've just released v7 stable. `npm view @babel/cli`
Might be a good idea to upgrade to the latest.